gradoop-common/src/test/java/org/gradoop/common/model/impl/properties/PropertyValueTest.java

```diff
@@ -88,12 +88,12 @@ public void testBigDecimalConversion() {
 
   @Test
   public void testFailedBigDecimalConversion() {
-    exception.expect(ClassCastException.class);
+    exception.expect(IllegalArgumentException.class);
```

Comment:
You can use `@Test(expected = IllegalArgumentException.class)` instead, since you only expect one exception in this test.